src/fswAlgorithms/stateEstimation/thrustCMEstimation/thrustCMEstimation.cpp

```diff
@@ -107,7 +108,7 @@ void ThrustCMEstimation::UpdateState(uint64_t CurrentSimNanos)
     if ((this->attGuidInMsg.isWritten()) && (attError < this->attitudeTol)) {
 
         /*! subtract torque about point B from measurement model */
-        y = L_B - r_TB_B.cross(T_B);
+        y = L_B - (r_TB_B.cross(T_B)) - LSwirl_B;
```

Comment:
What's the idea behind this change here? I'm admittedly unfamiliar with the thrust CME module.

Reply:
Just incorporating the swirl torque into the measurement model. Without swirl torque, the total thruster torque is just moment arm x thrust force (r_TB_T x T_B), now the swirl torque LSwirl_B is added too.

Reply:
I added a small note to make it explicit.